builder/builder-next/exporter/export.go

```diff
@@ -44,78 +46,109 @@ func New(opt Opt) (exporter.Exporter, error) {
 }
 
 func (e *imageExporter) Resolve(ctx context.Context, opt map[string]string) (exporter.ExporterInstance, error) {
-	i := &imageExporterInstance{imageExporter: e}
+	i := &imageExporterInstance{
+		imageExporter: e,
+		buildInfo:     true,
+	}
 	for k, v := range opt {
 		switch k {
 		case keyImageName:
-			for _, v := range strings.Split(v, ",") {
-				ref, err := distref.ParseNormalizedNamed(v)
-				if err != nil {
-					return nil, err
-				}
-				i.targetNames = append(i.targetNames, ref)
+			i.targetName = v
+		case keyBuildInfo:
+			if v == "" {
+				i.buildInfo = true
+				continue
+			}
+			b, err := strconv.ParseBool(v)
+			if err != nil {
+				return nil, errors.Wrapf(err, "non-bool value specified for %s", k)
```

Comment:
We need to have a look at the error-types returned (and add a proper "invalid parameter" type); perhaps we need to consider moving some of this paring into the api side, and use `httputils.BoolValue()` (or a variant thereof, as it feels a bit _too_ permissive for an API); https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/api/server/httputils/form.go#L10-L13

There's already a bunch of these in this package, so let's do so separately.